iotedgedev/envvars.py

```diff
@@ -118,7 +118,6 @@ def load(self, force=False):
                 self.get_registries()
 
                 self.BYPASS_MODULES = self.get_envvar("BYPASS_MODULES", default="")
-                self.ACTIVE_DOCKER_PLATFORMS = self.get_envvar("ACTIVE_DOCKER_PLATFORMS", altkeys=["ACTIVE_DOCKER_ARCH"], default="")
```

Comment:
Adding note for backwards compat issues when removing this.

Reply:
Thanks, we will add notes for this change.